locale/en/LC_MESSAGES/django.po

```diff
@@ -3506,15 +3507,15 @@ msgstr "Assistant grading is not allowed for this assignment."
 
 #: exercise/permissions.py
 msgid "SUBMISSION_VISIBILITY_PERMISSION_DENIED_MSG"
-msgstr "Unfortunately you are not permitted to see this submission."
+msgstr "Unfortunately you are not permitted to see view submission."
```

Comment:
typo
```suggestion
msgstr "Unfortunately you are not permitted to view this submission."
```

Usually, there is a comma after "unfortunately" when it starts a sentence.